addons/prometheusadapter/0.5.x/prometheusadapter-1.yaml

```diff
@@ -53,7 +53,7 @@ spec:
                  resource: namespace
                 pod_name:
```

Comment:
should this not also be renamed to `pod`? Basing off [these changes](https://github.com/helm/charts/pull/19961/files#diff-c22b7d7670f281dfec8cc6b348229a1eR77)

Reply:
Good catch!

Reply:
I pushed a fix to this branch